artifactory/commands/transferfiles/utils.go

```diff
@@ -28,6 +30,11 @@ var curThreads int
 
 type InterruptionErr struct{}
 
+type PollingChunk struct {
```

Comment:
PollingChunk --> FilesChunk

Let's also add the description for this struct, explaining what this strcut represents and what is its function.